dot/node_test.go

```diff
@@ -200,17 +201,75 @@ func TestNewNodeC(t *testing.T) {
 			},
 			err:  errors.New("no keys provided for authority node"),
 		},
-		// TODO this is commented out because in holds a lock on badger db, causing next test to foil
+		{
+			name: "minimal config",
+			args: args{
+				cfg: &Config{
+					Global:  GlobalConfig{BasePath: cfg.Global.BasePath,},
+					Init:    InitConfig{Genesis: genFile.Name()},
+					Account: AccountConfig{Key: "alice"},
+					Core:    CoreConfig{WasmInterpreter: wasmer.Name,},
+				},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := NewNodeC(tt.args.cfg)
+			if tt.err != nil {
+				assert.EqualError(t, err, tt.err.Error())
+				utils.RemoveTestDir(t)
```

Comment:
nit maybe use `t.TestDir` instead?

Reply:
I'm not sure I understand what you mean here?

Reply:
I'm not sure where the directory is created but you can use `t.TestDir` to create a temp directory that gets removed at the end of the test. It might be out of scope here though.